mlflow/java/spark/pom.xml

```diff
@@ -7,8 +7,11 @@
     <maven.compiler.source>1.8</maven.compiler.source>
     <maven.compiler.target>1.8</maven.compiler.target>
     <encoding>UTF-8</encoding>
-    <scala.version>2.12.6</scala.version>
-    <scala.compat.version>2.12</scala.compat.version>
+    <!-- Overriding Scala version to 2.13 for mlflow-spark.
+      We can't bump it up globally because mleap doesn't support scala 2.13 yet, which is dependency of mlflow-scoring.
+      Ref: https://github.com/combust/mleap/issues/811 -->
+    <scala.version>2.13.10</scala.version>
```

Comment:
I think we should still support scala 2.12, spark 3.5 still supports scala 2.12.

We can support to build mlflow-spark with either scala-2.12 or scala 2.13, like this project:

https://github.com/dmlc/xgboost/blob/3f4e22015a6796a0807650d7b2f97b108f8760df/jvm-packages/xgboost4j-spark/pom.xml#L12

and we can release 2 jars , one for scala 2.12, one for scala 2.13, like:
https://mvnrepository.com/artifact/ml.dmlc/xgboost4j-spark

WDYT ?

Reply:
Yeah I faced exact issue when calling from Pyspark, and was working exactly same approach:) 
Will push another revision soon!